test/cli.js

```diff
@@ -364,6 +364,14 @@ test('bails when config contains `"tap": true` and `"watch": true`', t => {
 	});
 });
 
+test('bails when config does not contain `--concurrency` and `-c` values', t => {
```

Comment:
These tests can be hard to get right. I'd start with running the CLI directly (`node cli.js test/fixture/concurrency/test.js` etc) to make sure it works correctly. Then try invoking it from the test.